src/App.js

```diff
@@ -1,20 +1,34 @@
-import './App.css';
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] It is always a good practice to make sure you fix all linter errors instead of disabling them, A linter is great for identifying errors when you use standard rules. Remember, a linter analyzes your code for stylistic and programming errors against the rules it knows. 💯 😉